components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivityBrowser.java

```diff
@@ -668,7 +721,43 @@ public boolean isReviewed(String targetId, String userId) throws SocialActivityE
         } finally {
             DSConnection.closeConnection(connection);
         }
-        return reviewed;
+        return commented;
     }
 
+    /**
+     * Get the total likes or dislikes for the given targetId(Review ID)
+     *
+     * @param targetId  Review ID
+     * @param likeValue Likevalue whether request total likes(1) count or dislike(0) counts
+     * @return Total number of likes/dislikes
+     * @throws SocialActivityException
+     */
+    private int getTotalLikes(int targetId, int likeValue) throws SocialActivityException {
```

Comment:
The formatting issue appears to be with the existing code.I have verified by checking with the wso2 IDE style